usr/lib/greenboot/check/wanted.d/01_update_platforms_check.sh

```diff
@@ -5,7 +5,7 @@ REPOS_DIRECTORY=/etc/ostree/remotes.d
 URLS_WITH_PROBLEMS=()
 
 get_update_platform_urls() {
-    mapfile -t UPDATE_PLATFORM_URLS < <(grep -P -ho 'http[s]?.*' "${REPOS_DIRECTORY}"/*)
+    mapfile -t UPDATE_PLATFORM_URLS < <(ostree remote list | xargs ostree remote show-url)
```

Comment:
fetching it from rpm-ostree directly makes sense but this actions needs to iterative as rpm-ostree can have multiple remotes that it can rebase off  [reference doc](https://redhat-internal.slack.com/archives/C999USB0D/p1695815913023619?thread_ts=1695815581.270249&cid=C999USB0D)